src/backend/api/reviews.js

```diff
@@ -0,0 +1,117 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+const reviewsColumns = new Set([
+  "title",
+  "description",
+  "meal_id",
+  "stars",
+  "created_date",
+]);
+
+// this function checks if input data valid and makes possible to update only neccesary data in table
+function checkTableData(table) {
```

Comment:
Nitpick(naming): This is the request input, not the table that is being checked :)